src/app/searchResults/searchResults.component.js

```diff
@@ -12,29 +12,52 @@ let componentName = 'searchResults';
 class SearchResultsController {
 
   /* @ngInject */
-  constructor(designationsService) {
+  constructor($window, $location, designationsService) {
+    this.$window = $window;
+    this.$location = $location;
     this.designationsService = designationsService;
+    this.searchParams = {};
   }
 
   $onInit(){
-    let searchTerm = urlQueryTerm.get('q');
-    if(searchTerm){
-      this.designationsService.createSearch(searchTerm)
-        .subscribe((id) => {
-          this.searchId = id;
-          this.getPageResults(1);
+    var params = this.$location.search();
+
+    this.searchParams.keyword = params.q;
+    this.searchParams.first_name = params.fName;
+    this.searchParams.last_name = params.lName;
```

Comment:
What's the purpose of copying these if `requestSearch()` copies them again? Do they need to be there even if the type isn't `people`?